lessons/basics/enum.md

```diff
@@ -12,9 +12,95 @@ A set of algorithms for enumerating over collections.
 
 ## Enum
 
-The `Enum` module includes over one hundred functions for working with the collections we learned about in the last lesson.
+The `Enum` module includes nearly 100 functions for working with the collections we learned about in the last lesson.
 
-This lesson will only cover a subset of the available functions, to see a complete set of functions visit the official [`Enum`](http://elixir-lang.org/docs/stable/elixir/Enum.html) docs; for lazy enumeration use the [`Stream`](http://elixir-lang.org/docs/stable/elixir/Stream.html) module.
+This lesson will only cover a subset of the available functions, however we can actually enamine them ourselves.
+Let's do a little experiment in IEx.
+
+```elixir
+$ iex
```

Comment:
Please drop the `$ iex` and the numbers from the `iex` prompt, we have a style guide that tries to capture some of these: https://github.com/doomspork/elixir-school/wiki/Lesson-Styleguide